src/Components/DropDown/utils.tsx

```diff
@@ -52,7 +63,29 @@ export const renderOption = (props: any, option: any) => {
           style={{ width: 24, height: 24, borderRadius: "50%" }}
         />
       )}
-      {option.label} {option.value}
+      <Typography type={TypographyTypes.BODY_REGULAR} label={option.label} />
     </Box>
   );
 };
+const styles = (theme: Theme) => ({
+  popper: {
+    maxWidth: "fit-content",
+  },
+});
+
+type OptionType = {
+  title: string;
+};
+
+interface PopperMyProps {}
+
+export const PopperMy: React.FC<PopperMyProps> = (props) => {
+  return (
+    <Popper
+      open={false}
+      {...props}
+      style={styles({} as Theme).popper}
+      placement="bottom-start"
+    />
+  );
+};
```

Comment:
this is not util funcrtion, this is a component, move it to another folder components inside of this directory.
also , change its name to a more understandable name.